go/libkb/leveldb.go

```diff
@@ -130,7 +130,7 @@ func NewLevelDb(g *GlobalContext, filename func() string) *LevelDb {
 		Contextified: NewContextified(g),
 		filename:     path,
 		dbOpenerOnce: new(sync.Once),
-		cleaner:      newLevelDbCleaner(g, filepath.Base(path)),
+		cleaner:      newLevelDbCleaner(NewMetaContext(context.TODO(), g), filepath.Base(path)),
```

Comment:
There are `NewMetaContextTODO(g)` and `NewMetaContextBackground(g)` functions btw

Reply:
ah nice